pkg/bufwriter/writer.go

```diff
@@ -0,0 +1,28 @@
+// Package bufwriter provides a wrapper for io.WriteCloser that uses
+// buffered output via a bufio.Writer and calls Flush on close.
+// Not clear why bufio.Writer doesn't do this.
+package bufwriter
+
+import (
+	"bufio"
+	"io"
+)
+
+type Writer struct {
+	io.WriteCloser
+	writer *bufio.Writer
+}
+
+func New(w io.WriteCloser) io.WriteCloser {
```

Comment:
Nit: The Go idiom is to return a concrete type (a *Writer in this case) rather than an interface.

Reply:
+1